app/Http/Resources/SaveResourceDescriptorResource.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class SaveResourceDescriptorResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+        return [
+            "id" => $this->id,
```

Comment:
same here. can later removed.

For more see: https://trello.com/c/Bwu2JG1K/458-ressourcenport-im-backend-hinzuf%C3%BCgen#comment-651f13a7c290023389d18a96